tests/pytests/test_vecsim.py

```diff
@@ -322,15 +322,16 @@ def test_with_fields(env):
     conn.execute_command('FT.CREATE', 'idx', 'SCHEMA', 'v', 'VECTOR', 'HNSW', '6', 'TYPE', 'FLOAT32', 'DIM', dimension, 'DISTANCE_METRIC', 'L2', 't', 'TEXT')
     load_vectors_into_redis(conn, 'v', dimension, qty)
 
-    query_data = np.float32(np.random.random((1, dimension)))
-    res = env.cmd('FT.SEARCH', 'idx', '5 @v:[$vec_param TOPK 100]',
-                    'SORTBY', 'v_score', 'PARAMS', 2, 'vec_param', query_data.tobytes(),
-                    'RETURN', 2, 'v_score', 't')
-    res_nocontent = env.cmd('FT.SEARCH', 'idx', '5 @v:[$vec_param TOPK 100]',
-                    'SORTBY', 'v_score', 'PARAMS', 2, 'vec_param', query_data.tobytes(),
-                    'NOCONTENT')
-    env.assertEqual(res[1::2], res_nocontent[1:])
-    env.assertEqual('t', res[2][2])
+    for _ in env.retry_with_rdb_reload():
+        query_data = np.float32(np.random.random((1, dimension)))
+        res = env.cmd('FT.SEARCH', 'idx', '5 @v:[$vec_param TOPK 100]',
+                        'SORTBY', 'v_score', 'PARAMS', 2, 'vec_param', query_data.tobytes(),
+                        'RETURN', 2, 'v_score', 't')
+        res_nocontent = env.cmd('FT.SEARCH', 'idx', '5 @v:[$vec_param TOPK 100]',
+                        'SORTBY', 'v_score', 'PARAMS', 2, 'vec_param', query_data.tobytes(),
+                        'NOCONTENT')
+        env.assertEqual(res[1::2], res_nocontent[1:])
+        env.assertEqual('t', res[2][2])
```

Comment:
can you add an additional assert with `FT.DEBUG` over the vector field? compare the parameters to the info there?